shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-spring/shardingsphere-sharding-spring-namespace/src/test/java/org/apache/shardingsphere/sharding/spring/namespace/ShardingSpringNamespaceTest.java

```diff
@@ -105,141 +103,141 @@
     }
     
     @Test
     public void assertComplexStrategy() {
-        assertThat(complexStrategy.getShardingColumns(), is("order_id,user_id"));
-        assertThat(complexStrategy.getShardingAlgorithmName(), is("complexShardingAlgorithm"));
+        Assert.assertThat(complexStrategy.getShardingColumns(), CoreMatchers.is("order_id,user_id"));
+        Assert.assertThat(complexStrategy.getShardingAlgorithmName(), CoreMatchers.is("complexShardingAlgorithm"));
     }
     
     @Test
     public void assertHintStrategy() {
-        assertThat(hintShardingStrategy.getShardingAlgorithmName(), is("hintShardingAlgorithm"));
+        Assert.assertThat(hintShardingStrategy.getShardingAlgorithmName(), CoreMatchers.is("hintShardingAlgorithm"));
     }
     
     @Test
     public void assertNoneStrategy() {
-        assertNotNull(noneStrategy);
+        Assert.assertNotNull(noneStrategy);
     }
     
     @Test
     public void assertIncrementAlgorithm() {
-        assertThat(incrementAlgorithm.getType(), is("INCREMENT"));
+        Assert.assertThat(incrementAlgorithm.getType(), CoreMatchers.is("INCREMENT"));
     }
     
     @Test
     public void assertDefaultKeyGenerator() {
-        assertThat(defaultKeyGenerator.getColumn(), is("id"));
-        assertThat(defaultKeyGenerator.getKeyGeneratorName(), is("incrementAlgorithm"));
+        Assert.assertThat(defaultKeyGenerator.getColumn(), CoreMatchers.is("id"));
+        Assert.assertThat(defaultKeyGenerator.getKeyGeneratorName(), CoreMatchers.is("incrementAlgorithm"));
     }
     
     @Test
     public void assertOrderKeyGenerator() {
-        assertThat(orderKeyGenerator.getColumn(), is("order_id"));
-        assertThat(orderKeyGenerator.getKeyGeneratorName(), is("incrementAlgorithm"));
+        Assert.assertThat(orderKeyGenerator.getColumn(), CoreMatchers.is("order_id"));
+        Assert.assertThat(orderKeyGenerator.getKeyGeneratorName(), CoreMatchers.is("incrementAlgorithm"));
     }
     
     @Test
     public void assertSimpleRule() {
         Collection<ShardingTableRuleConfiguration> actualSimpleRuleConfigurations = simpleRule.getTables();
-        assertThat(actualSimpleRuleConfigurations.size(), is(1));
+        Assert.assertThat(actualSimpleRuleConfigurations.size(), CoreMatchers.is(1));
         ShardingTableRuleConfiguration actualSimpleRuleConfiguration = actualSimpleRuleConfigurations.iterator().next();
-        assertThat(actualSimpleRuleConfiguration.getLogicTable(), is("t_order"));
+        Assert.assertThat(actualSimpleRuleConfiguration.getLogicTable(), CoreMatchers.is("t_order"));
     }
     
     @Test
     public void assertComplexRule() {
         Collection<ShardingTableRuleConfiguration> actualComplexRuleConfigurations = complexRule.getTables();
-        assertThat(actualComplexRuleConfigurations.size(), is(1));
+        Assert.assertThat(actualComplexRuleConfigurations.size(), CoreMatchers.is(1));
```

Comment:
Please import `CoreMatchers`, so that we can change all the `CoreMatchers.is` as `is`.

Reply:
It will cause code check error. I have be suggested to avoid startic import.